### ISSUE TITLE
add a recommended preset

### ISSUE THREAD

**Issue by Aladdin-ADD** [gajus/eslint-plugin-jsdoc#49]

**Comment by rjhilgefort** [comment]
👍 I was coming into make this exact issue. I think that's pretty idiomatic amongst `eslint` plugins.

**Comment by pdehaan** [comment]
This seems pretty simple... What about the following recommended config defaults?

```js
  configs: {
    recommended: {
      rules: {
        'jsdoc/check-param-names': 'warn',
        'jsdoc/check-tag-names': 'warn',
        'jsdoc/check-types': 'warn',
        'jsdoc/newline-after-description': 'warn',
        'jsdoc/require-description-complete-sentence': 'off',
        'jsdoc/require-example': 'off',
        'jsdoc/require-hyphen-before-param-description': 'off',
        'jsdoc/require-param': 'warn',
        'jsdoc/require-param-description': 'warn',
        'jsdoc/require-param-name': 'warn',
        'jsdoc/require-param-type': 'warn',
        'jsdoc/require-returns-description': 'warn',
        'jsdoc/require-returns-type': 'warn'
      }
    }
  }
```

Should be easy enough for people to convert warnings to errors if they want in their projects. Or, I could also add a "plugin:jsdoc/strict" config which sets everything to "error", but maybe that's too far.
